packages/venia-ui/lib/components/PageLoadingIndicator/indicator.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import defaultClasses from './indicator.css';
+import { mergeClasses } from '../../classify';
+import { RotateCw as LoaderIcon } from 'react-feather';
+import Icon from '../Icon';
+
+const PageLoadingIndicator = props => {
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    return (
+        <div className={classes.root}>
+            <Icon
+                src={LoaderIcon}
+                size={24}
+                classes={{ icon: classes.indicator }}
```

Comment:
I moved the animation onto the `.root` style, but we still require the `color` style to be applied to the `<Icon />`

Reply:
Understood. Instead of overriding `icon` here, probably override `root`, and supply `--stroke` instead of `color`. Make sure `--stroke` is in `rgb(x, y, z)` format.